tools/gcpviz/.gitlab-ci.yml

```diff
@@ -0,0 +1,27 @@
+defaults: 
```

Comment:
Was this file left here by mistake ? We don't use gitlab in our repo

Reply:
No, it's a sample pipeline for Gitlab. I'll rename to it to gitlab-ci.yml so it's more noticiable